utils/src/main/java/software/amazon/awssdk/utils/IoUtils.java

```diff
@@ -81,13 +88,26 @@ public static void closeQuietly(AutoCloseable is, Logger log) {
      * Closes the given Closeable quietly.
      * @param maybeCloseable the given closeable
      * @param log logger used to log any failure should the close fail
+     * @deprecated use {@link #closeQuietlyV2(AutoCloseable, Logger)} instead
      */
-    public static void closeIfCloseable(Object maybeCloseable, Logger log) {
+    @Deprecated
+    public static void closeIfCloseable(Object maybeCloseable, org.slf4j.Logger log) {
         if (maybeCloseable instanceof AutoCloseable) {
             IoUtils.closeQuietly((AutoCloseable) maybeCloseable, log);
         }
     }
 
+    /**
+     * Closes the given Closeable quietly.
+     * @param maybeCloseable the given closeable
+     * @param log logger used to log any failure should the close fail
+     */
+    public static void closeIfCloseableV2(Object maybeCloseable, Logger log) {
```

Comment:
We can't use the same signature because it'll break people who invoke `closeIfClosable(closable, null)`